pkg/istio-agent/sds-agent.go

```diff
@@ -73,6 +74,9 @@ var (
 
 	// Location of K8S CA root.
 	k8sCAPath = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+
+	// CitadelCACertPath is the directory for Citadel CA certificate.
+	CitadelCACertPath = "/etc/istio/citadel-ca-cert"
```

Comment:
1. Please use ./ - so it works on VMS and not-root environments ( it's equivalent in k8s since we run in /)

2. Please use /var/lib/istio/ca-certs - most of the configs are there, and we don't want to include citadel name ( it doesn't have to be a citadel root anyways ).

Reply:
A good location is: "./var/lib/istio/certs/ca-certificates.crt" ( similar with std cert file ), or /var/lib/istio/ca-certificates if we are going to provide individual files.

Reply:
This is to consistent with the configuration in istio/installer repository. There is another well known path for CA root certificate under ./etc/certs/root-cert.pem.